gammapy/maps/wcs/tests/test_ndmap.py

```diff
@@ -1045,3 +1045,18 @@ def test_plot_mask():
 
     with mpl_plot_check():
         mask.plot_grid()
+
+
+def test_histogram_center_value():
+    evt_energy = np.arange(100.0, 105, 0.5) * u.GeV
+    N_evt = evt_energy.shape[0]
+    center = SkyCoord(0, 0, unit="deg", frame="icrs")
+    radec = SkyCoord(
+        [center.ra] * N_evt, [center.dec] * N_evt, unit="deg"
+    )  # fake position list
```

Comment:
Could you try with different energies? Also to check that spatially this behaves as expected. Maybe you could take pix values and also test `fill_by_pix`

Reply:
added a `fill_by_pix`